src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,41 @@
 import React from 'react';
+import { Post } from '../../types/Post';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+import './PostInfo.scss';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+interface Props {
+  post: Post;
+}
+
+export const PostInfo: React.FC<Props> = ({ post }) => {
+  const {
+    title,
+    user,
+    body,
+    comments,
+  } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{title}</h3>
+
+        {user && (
+          <p>
+            {' Posted by  '}
+            {user && <UserInfo user={user} />}
```

Comment:
you already check it
```suggestion
            <UserInfo user={user} />
```